js/zb.js

```diff
@@ -146,6 +146,9 @@ module.exports = class zb extends Exchange {
                     'public': 'https://api.zb.work/data',
                     'private': 'https://trade.zb.work/api',
                     'trade': 'https://trade.zb.work/api',
+                    'swapV1': 'https://fapi.zb.com/api/public',
+                    'swapV2': 'https://fapi.zb.com/Server/api',
+                    'swapV2Private': 'https://fapi.zb.com/Server/api',
```

Comment:
Thanks for the explanation, I like this design a lot better, I'll start adjusting this and let you know if I have any further questions

Reply:
@Dan-krm thx! Let me know if you run into issues, will do assist as much as I can.

Reply:
I removed the trade key like you mentioned but I noticed that the handling of trade and private was different in sign. Since trade only had one corresponding path 'getFeeInfo' I'm currently handling this in sign by checking for the path.

I'm unsure of the best way to test my implementation of private contract V2 in sign (lines 1286 - 1297) without creating a test method, I would appreciate an example of the best approach for doing that if possible.

Reply:
You can do that with the implicit API using `cli.js`:

- https://github.com/ccxt/ccxt/wiki/Manual#implicit-api

To see all methods for the implicit swap API:

```
node examples/js/cli zb | grep swap
```

Then to call an implicit method:

```
node examples/js/cli zb swapV2PrivateGet...
```

Reply:
Thanks I've used this in the past for public methods, the grep command and exact link in the manual will be helpful